### ISSUE TITLE
Error: "SignInWithApple.authorize()" is not implemented on android

### ISSUE THREAD

**Issue by Wallfacer005CN** [capacitor-community/apple-sign-in#107]
**Describe the bug**
Error: "SignInWithApple.authorize()" is not implemented on android

Although the readme document states that Android is not supported, I noticed that there is Android code in the repository, and the update log shows that Android has been supported from a certain version. However, when I tested it on an Android device, I encountered an error indicating that it is not supported.

**Comment by piotr-cz** [comment]
Current Android code is just a [Capacitor plugin placeholder](https://github.com/capacitor-community/apple-sign-in/blob/v5.0.0/android/src/main/java/com/getcapacitor/community/applesignin/SignInWithApple.java) (that hasn't been updated since Capacitor v3: #105).

This may be fixed either by:
- implementing #12
- removing placeholder and using Web version on Android platform

**Comment by Wallfacer005CN** [comment]
> Current Android code is just a [Capacitor plugin placeholder](https://github.com/capacitor-community/apple-sign-in/blob/v5.0.0/android/src/main/java/com/getcapacitor/community/applesignin/SignInWithApple.java) (that hasn't been updated since Capacitor v3: #105).
> 
> This may be fixed either by:
> 
> * implementing [Add android support #12](https://github.com/capacitor-community/apple-sign-in/pull/12)
> * removing placeholder and using Web version on Android platform

Thanks very much for your reply.
This is my first time  using Capacitor and this plugin.
Could you please tell me how to “removing placeholder and using Web version on Android platform” ？

**Comment by piotr-cz** [comment]
@Wallfacer005CN 
This must be implemented in a pull request.

At this moment I'm not showing Sign in with Apple only on iOS platform like this:

```tsx
import { isPlatform } from '@ionic/react'

const authenticationServices: JSX.Element[] =
  isPlatform('ios') && isPlatform('capacitor')
    ? [<AppleSignIn />, <GoogleLogin />]
    : [<GoogleLogin />]

// .. render available authentication services
```

**Comment by Wallfacer005CN** [comment]
> @Wallfacer005CN This must be implemented in a pull request.
> 
> At this moment I'm not showing Sign in with Apple only on iOS platform like this:
> 
> ```tsx
> import { isPlatform } from '@ionic/react'
> 
> const authenticationServices: JSX.Element[] =
>   isPlatform('ios') && isPlatform('capacitor')
>     ? [<AppleSignIn />, <GoogleLogin />]
>     : [<GoogleLogin />]
> 
> // .. render available authentication services
> ```

Thanks very much.
I will try it.

**Comment by Wallfacer005CN** [comment]
> @Wallfacer005CN This must be implemented in a pull request.
> 
> At this moment I'm not showing Sign in with Apple only on iOS platform like this:
> 
> ```tsx
> import { isPlatform } from '@ionic/react'
> 
> const authenticationServices: JSX.Element[] =
>   isPlatform('ios') && isPlatform('capacitor')
>     ? [<AppleSignIn />, <GoogleLogin />]
>     : [<GoogleLogin />]
> 
> // .. render available authentication services
> ```



> @Wallfacer005CN This must be implemented in a pull request.
> 
> At this moment I'm not showing Sign in with Apple only on iOS platform like this:
> 
> ```tsx
> import { isPlatform } from '@ionic/react'
> 
> const authenticationServices: JSX.Element[] =
>   isPlatform('ios') && isPlatform('capacitor')
>     ? [<AppleSignIn />, <GoogleLogin />]
>     : [<GoogleLogin />]
> 
> // .. render available authentication services
> ```

Thank you very much.
It looks that I  misunderstood.

**Comment by Bhagatameya21** [comment]
Same Issue No Solution for Android atm?

**Comment by WcaleNieWolny** [comment]
Hello, although not in this plugin, I implemented Apple login on Android here: https://github.com/Cap-go/capacitor-social-login

You can find the docs for it: [here](https://github.com/Cap-go/capacitor-social-login/blob/main/docs/setup_apple.md#apple-login-on-android)

**Comment by piotr-cz** [comment]
@WcaleNieWolny 

You mean that https://github.com/Cap-go/capacitor-social-login is not just an Apple login plugin for capacitor, but a plugin that handles login for multiple providers (Google, Apple, Facebook) ?

**Comment by WcaleNieWolny** [comment]
Correct, but you don't have to use other providers. Capgo Social Login handles both Apple and Google with excellent documentation. As for Facebook, it was added my by coworker and it's but currently documented, but from what I know it should work
